apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/ContextManagerExtendService.java

```diff
@@ -27,8 +27,13 @@
 import org.apache.skywalking.apm.agent.core.remote.GRPCChannelStatus;
 import org.apache.skywalking.apm.agent.core.sampling.SamplingService;
 
+import java.util.Arrays;
+
 @DefaultImplementor
 public class ContextManagerExtendService implements BootService, GRPCChannelListener {
+    
+    private static String[] IGNORE_SUFFIX_ARRAY = Config.Agent.IGNORE_SUFFIX.split(",");
```

Comment:
Put the initialization process in the `boot` method, rather than there. It is safer and keeps the initialization sequence more clear.